From bd53e2fcd2ff2ab5f9726753b827e7f958ad1041 Mon Sep 17 00:00:00 2001
From: dpapad <dpapad@chromium.org>
Date: Wed, 27 Mar 2024 23:05:52 +0000
Subject: [PATCH] Backport "NewTabPage: Add missing dep to
 cr_components/history_clusters/."

This fixes transient build error caused by missing dependency.
The patch will be included in upstream's 125 release.

Upstream-Status: Backport [https://crrev.com/c/5402989]
Signed-off-by: Daniel Semkowicz <dse@thaumatec.com>
---
 chrome/browser/resources/new_tab_page/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/resources/new_tab_page/BUILD.gn b/chrome/browser/resources/new_tab_page/BUILD.gn
index db4708a51c522..96612c978bcb6 100644
--- a/chrome/browser/resources/new_tab_page/BUILD.gn
+++ b/chrome/browser/resources/new_tab_page/BUILD.gn
@@ -97,6 +97,7 @@ build_webui("build") {
     "//ui/webui/resources/cr_components/color_change_listener:build_ts",
     "//ui/webui/resources/cr_components/customize_themes:build_ts",
     "//ui/webui/resources/cr_components/help_bubble:build_ts",
+    "//ui/webui/resources/cr_components/history_clusters:build_ts",
     "//ui/webui/resources/cr_components/most_visited:build_ts",
     "//ui/webui/resources/cr_components/omnibox:build_ts",
     "//ui/webui/resources/cr_components/page_image_service:build_ts",
